docs/layers.md

```diff
@@ -1514,6 +1515,35 @@ The `hgv` property indicates general truck heavy goods vehicle truck access, val
 
 For `hgv_restriction` property indicates general truck heavy goods vehicle truck access restrictions, values (units vary) include: `weight` (metric tonnes), `height` (metres), `length` (metres), `width` (metres), `wpa` (weight per axle, in metric tonnes), `kpra` (king pin to rear axle length, in metric tonnes), `hazmat` (true if restricted, otherwise omitted), `axles` (number of axles), `other` and `multiple` if more than one.
 
+#### Road Transportation `hgv_time_restrictions` Values
+Time restriction is a semicolon-delimited array of date and time restrictions,
+where date and time of a restriction are delimited with a pipe `|` character. For example,
+the restriction "Monday, Tuesday, Friday from 7 to 20; Saturday, Sunday from
+dusk to dawn" is given as follows:
+
+```
+days_of_week Mon,Tue,Fri|start_time25200end_time72000;days_of_week Sat,Sun|dusk_to_dawn
+```
+
+* `days_of_week` - array of weekdays, for example, `days_of_week Mon,Tue,Fri`
+* `date_range` - start and end dates as unix timestamps, for example, `date_range 1546344000 1577880000`
+* `days_of_month` - range of days of month, for example, `days_of_month 1-10`
+* `day_week_month` - range of days of week of month, for example, `day_week_month Mon1-Sat3`
+* `day_week_year` - range of days of week of year, for example, `day_week_year Wed12-Fri36`
+* `week_of_month` - range of weeks of month, for example, `week_of_month 1-4`
+* `month_of_year` - range of months of year, for example, `month_of_year Jan-Jul`
+* `day_month_year` - range of days of months of year, for example, `day_month_year Jan1-Jul31`
+* `day_week_month_year` - range of days of weeks of months of year, for example, `day_week_month_year JulSun1-DecMon1`
+* `externally_specified` - externally specified string value, for example, `externally_specified <string-value>`
```

Comment:
You mean this is a link to a web address or raw text or?

Reply:
@rinnyB can you resolve this one and then the PR will be in mergable state, please?